topostats/io.py

```diff
@@ -840,49 +885,52 @@ def load_gwy(self) -> tuple:
             image = None
             has_image_found = False
             units = ""
-
-            re = r"\/(\d+)\/data$"
-            components = list(image_data_dict.keys())
-            conv_factors = {"m": 1e9, "um": 1e6, "mm": 1e3}
-            for component in components:
-                match = regex.match(re, component)
-                if match == None:
+            # How can we get current configuration["loading"] ?
+            # default_config.yaml is used to get conf["loading"]["scale"]
+            default_config = pkg_resources.open_text(__package__, "default_config.yaml").read()
```

Comment:
Rather than loading the config again we have structured classes to take arguments and use the same argument name in the YAML configuration and the parameters are passed in using Pythons `**kwargs` functionality as the names match.

Thus to get the `loading` dictionary we need to add an argument to the `__init__` method of the `LoadScans` and on [line 123](https://github.com/AFM-SPM/TopoStats/blob/dfbae6e56714414db3387f4b2a0923788dc88427/topostats/run_topostats.py#L123) of `topostats/run_topostats.py` this is loaded automatically and instead the line below can be modified to extract the scale.